tests/e2e/cjs/index.test.js

```diff
@@ -277,3 +290,37 @@ describe('ANTRegistry', async () => {
     assert(Array.isArray(affiliatedAnts.Controlled));
   });
 });
+
+describe('Signing', async () => {
+  const signers = [
+    new ArweaveSigner(testWallet),
+    createAoSigner(new ArweaveSigner(testWallet)),
+  ];
+
+  it('Should be able to sign on the IO contract with all ContractSigner types', async () => {
+    for (const signer of signers) {
+      const io = IO.init({ signer });
+
+      assert(io instanceof IOWriteable);
+    }
+  });
```

Comment:
move under `describe("IO"` and tests are typically lower case (e.g. `it('should be able to sign..',`